test/static_fixtures/templates/provision/redhat_kexec.erb

```diff
@@ -0,0 +1,60 @@
+<%#
```

Comment:
This is for `foreman_discovery` plugin right? If yes than the file should be in the plugin repository, not here

Reply:
I fixed it. Sorry for the misunderstanding.